hw/mcu/dialog/da1469x/src/da1469x_clock.c

```diff
@@ -547,6 +556,21 @@ da1469x_clock_lp_freq_get(void)
 #endif
 }
 
+static void
+da1469x_clock_lp_set_rtc_divs(void)
+{
+    /* Please see section 2.20.4 for details */
```

Comment:
Could you be more specific what section 2.20.4 is?

Reply:
Sure.

Reply:
Done.

Reply:
I could not find "DA1469x design doc" anywhere are you sure it is available?

Reply:
The actual file name is "Lithium_Dialog_Design_Documentation_RevE.pdf". I don't remember where I got it from; I believe it may have been DM'd to me.

Reply:
this is described in section 34.3 of DA1469x datasheet so please update comment accordingly

Reply:
OK, I see... looks like the design doc was fashioned into the actual datasheet. Comment updated.

Reply:
Actually, the Lithium Design Doc is specific to our version of the MCU - however, the features mynewt-core covers apply to DA1469x, so referencing that datasheet is indeed more appropriate.